packages/beacon-node/src/network/processor/index.ts

```diff
@@ -269,15 +264,6 @@ export class NetworkProcessor {
           });
           return;
         }
-        if (
-          slot === clockSlot &&
-          (topicType === GossipType.beacon_block ||
-            topicType === GossipType.blob_sidecar ||
-            topicType === GossipType.data_column_sidecar)
-        ) {
-          // in the worse case if the current slot block is not valid, this will be reset in the next slot
-          this.isProcessingCurrentSlotBlock = true;
-        }
```

Comment:
@twoeths I removed this during merge conflict resolution, based on https://github.com/ChainSafe/lodestar/pull/7848 I think we no longer want it

Reply:
yes that's correct